kw/platform/settings.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+
+VALID_USER_AGENT_RE = os.getenv(
+    "KIWI_VALID_USER_AGENT_RE",
+    (
+        r"^(?P<name>\S.+?)\/(?P<version>\S.+?) "
+        r"\(Kiwi\.com (?P<environment>\S.+?)\)(?: ?(?P<system_info>.*))$"
+    ),
+)
```

Comment:
I think this shouldn't be configurable via env var 🤔(it feels like that goes against the company standards commited as code philosophy.)

I'd put it in `kw.platform.constants`.

Reply:
And you mean all of these should be constants? If's funny I just made it confirugable via env variables in the last revision because I kind of had a feeling we were talking about making this configurable for testing :-D But anyway, it makes sense, is what I'm saying :-)

Reply:
The issue actually described this, see #7 — the datetimes should be configurable, but I don't see a reason the regex should also be configurable for live tests.